maven-release-api/src/main/java/org/apache/maven/shared/release/config/ReleaseDescriptor.java

```diff
@@ -567,6 +567,15 @@
      */
     String getAutoResolveSnapshots();
 
+    /**
+     * Get the line separator to use in the pom.xml.
+     *
+     * (https://issues.apache.org/jira/browse/MRELEASE-899).
```

Comment:
This is reduandant since it is in the commit message and blame.

Reply:
Fixed with https://github.com/apache/maven-release/pull/62/commits/349fe933a951cf1a879ac16d965c845784466397